param/__init__.py

```diff
@@ -1335,8 +1335,11 @@ def get_range(self):
         Only classes from modules that have been imported are added
         (see concrete_descendents()).
         """
-        classes = concrete_descendents(self.class_)
-        d=OrderedDict((name,class_) for name,class_ in classes.items())
+        classes = self.class_ if isinstance(self.class_, tuple) else (self.class_,)
```

Comment:
Is this the same as `classes = tuple(self.class_)`?  If so this simpler version seems more readable and clear to me.

Reply:
It's not the same no.